packages/cli/src/tests/register.test.js

```diff
@@ -178,10 +178,12 @@ describe('RegisterCommand', () => {
         .reply(201, exportedApp);
     }
 
-    fs.writeFileSync(
-      APP_RC_FILE,
-      `{"id":${privateApp.id},"key":"App${privateApp.id}"}`,
-    );
+    beforeEach(function () {
+      fs.writeFileSync(
+        APP_RC_FILE,
+        `{"id":${privateApp.id},"key":"App${privateApp.id}"}`,
+      );
+    });
```

Comment:
This wasn't wrapped in `beforeEach()` and thus always executed, even without the suite included and thus not cleaning up the file in its `after()`.